core/src/main/java/org/codice/imaging/nitf/core/common/impl/DateTimeImpl.java

```diff
@@ -12,103 +12,105 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  */
-package org.codice.imaging.nitf.core.common;
+package org.codice.imaging.nitf.core.common.impl;
 
 import java.time.ZoneId;
 import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
-import static org.codice.imaging.nitf.core.common.CommonConstants.NITF21_DATE_FORMAT;
+
+import org.codice.imaging.nitf.core.common.DateTime;
 
 /**
-    Date / time representation.
-*/
-public class DateTime {
+ * Date / time representation.
+ */
+public class DateTimeImpl implements org.codice.imaging.nitf.core.common.DateTime {
+    /**
+     * Create a DateTime instance for the current time.
+     *
+     * @return initialised DataTime instance.
```

Comment:
Fixed the typo inherited from https://github.com/codice/imaging-nitf/blob/master/core/src/main/java/org/codice/imaging/nitf/core/common/DateTime.java